cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunTestsTask.java

```diff
@@ -480,4 +440,38 @@ private void cleanTempCache(Project project, Path cachesRoot) {
         }
         return moduleCoverageMap;
     }
+
+    private String getClassPath() {
+        List<Path> dependencies = new ArrayList<>();
+        dependencies.add(ProjectUtils.getBallerinaRTJarPath());
+        dependencies.addAll(ProjectUtils.testDependencies());
+        StringJoiner cp = new StringJoiner(File.pathSeparator);
```

Comment:
Shall we use a meaningful variable name?

Reply:
Changed variable name for String joiner